release/gcb/generate_manifest.sh

```diff
@@ -59,15 +59,19 @@ function create_manifest_check_consistency() {
     echo "ISTIO_REPO_SHA:$ISTIO_REPO_SHA API_REPO_SHA:$API_REPO_SHA PROXY_REPO_SHA:$PROXY_REPO_SHA CNI_REPO_SHA:$CNI_REPO_SHA some shas not found"
     exit 8
   fi
+  popd
+  checkout_code "api" "${API_REPO_SHA}" .
```

Comment:
it's a bit confusing that the API_REPO_SHA is just the name of the directory and doesn't have any significance. Or did I understand it wrong?

Reply:
API_REPO_SHA is short sha which we get from go.mod file.
we use it to checkout api repo and get long sham which we can use in manifest file later on.